lrmd/lrmd.c

```diff
@@ -1321,7 +1322,25 @@ lrmd_rsc_execute_service_lib(lrmd_rsc_t * rsc, lrmd_cmd_t * cmd)
         }
     }
 
-    if (cmd->isolation_wrapper) {
+    if (safe_str_eq(rsc->class, "alert")) {
+        /* In the case of Alert, lrmd always set rsc->type from CRM_alert_path parameter. */
+        void *value_lookup = g_hash_table_lookup(params_copy, CRM_ALERT_KEY_PATH);
+        if (value_lookup != NULL) { 
+            action = services_action_create_generic((char*)value_lookup, NULL);
+            action->rsc = strdup(rsc->rsc_id);
+            action->action = strdup(cmd->action);
+            action->timeout = cmd->timeout;
+            action->standard = strdup("event");
+            action->id = strdup(rsc->rsc_id);
+            action->agent = strdup((char*)value_lookup);
+            action->params = params_copy;
```

Comment:
I matched it with Alert of crmd, but examine it again.
I think that all parameters are not necessary.